lib/java/com/google/android/material/snackbar/BaseTransientBottomBar.java

```diff
@@ -773,7 +773,9 @@ public void run() {
 
       // Set view to INVISIBLE so it doesn't flash on the screen before the inset adjustment is
       // handled and the enter animation is started
-      view.setVisibility(View.INVISIBLE);
+      if (view.getParent() != null) {
```

Comment:
This will never be executed give L764. 

Shouldn't we just switch the order of L776 and L777 to fix the issue?